Cargo.lock

```diff
@@ -2285,18 +2284,18 @@ checksum = "19d374276b40fb8bbdee95aef7c7fa6b5316ec764510eb64b8dd0e2ed0d7e7f5"
 
 [[package]]
 name = "crc32c"
-version = "0.6.4"
+version = "0.6.5"
```

Comment:
This causes the toolchain upgrade. See https://github.com/zowens/crc32c/issues/58.